app/controllers/comments_controller.rb

```diff
@@ -59,6 +62,8 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+
```

Comment:
great job remembering to add format.js to your methods, this is a nitpik, but remember to be mindful of white spaces, removing any unnecessary space